File: problemas/ad-hoc/2381/2381.lua
entrada = {}
for numero in io.read():gmatch("%S+") do
  entrada[#entrada + 1] = numero
end
n = tonumber(entrada[1], 10)
k = tonumber(entrada[2], 10)

alunos = {}
for _ = 1, n do
  table.insert(alunos, io.read())
end

table.sort(alunos)

print(alunos[k])

File: problemas/ad-hoc/2381/2381.clj
(require '[clojure.java.io :as io])
(require '[clojure.string :as str])

(let [entrada (vec (line-seq (io/reader *in*)))]
  (let [entrada2 (map read-string (str/split (nth entrada 0) #" "))
        n (nth entrada2 0)
        k (nth entrada2 1)
        alunos (sort (vec (subvec entrada 1 (count entrada))))]
    (println (nth alunos (- k 1)))))
